test/addresses/optimism.ts

```diff
@@ -0,0 +1,19 @@
+import { ADDRESSES } from '@oasisdex/oasis-actions/src/helpers/addresses'
+
+export const optimismAddresses = {
```

Comment:
Don't we have a single place with all the needed addresses? Why add a new place again ?

Reply:
In dev there's already test/addresses/mainnet.ts so this is just following on from that pattern is essentially flattening some of the addresses that come the ADDRESSES mapping